### ISSUE TITLE
Mudança de layout do site 🚀 

### ISSUE THREAD

**Issue by huandrey** [OpenDevUFCG/opendevufcg.org#160]
**Descreva a feature em detalhes**

A mudança de layout do site já vem sendo discutida há muito tempo, muitas coisas já foram sugeridas, mas tudo acabou ficando no campo das ideias. Fiquei muito hypado com alguns comentários feitos na reunião de apresentação e acabei fazendo um esboço do que pra mim seria uma boa a se fazer no site. 

**Necessidade da feature:**

O site hoje apresenta muito texto e acaba sendo cansativo. Além disso, a tarefa de mudar a cara do site seria um bom começo para a galera que quer colaborar e não tem muito conhecimento do mundo web.

**Como ela pode ser implementada:**

O maior desafio seria na parte de css e na escolha adequada e concisa dos textos a serem colocados no site. A cor que eu tentei reproduzir nesse esboço foi feito com uma gambiarra rsrs, mas basicamente coloquei um elipse _**color(#343587)**_ com um pouco de blur num fundo **_color(#151515)_**.

Vou deixar o link para o figma aqui, caso queiram dar uma olhada melhor, lá vocês podem copiar o projeto e editar na sua conta (eu acho). Fora isso, o figma disponibiliza algumas propriedades do css (clicando no objeto e indo em inspect). 

-> [CLIQUE AQUI (FIGMA)](https://www.figma.com/file/C6TNGUIMwQiL3TpEBJl4CJ/Untitled?node-id=0%3A1`)

Algumas ideias foram sugeridas no discord
1. [Renan](https://github.com/JoseRenan) - Adicionar seções extras no site + botão para direcionar para alguma página exclusiva;
2. [Juan](https://github.com/JuanBarros2) - Adicionar um overview dos membros (tecnologias que domina, atuação do membro na comunidade, etc);
3.  [Mateus](https://github.com/mateusoliveira2) - Fazer meio que um setInterval nas fotos dos membros, seria bacana pra não ter uma tela inteira de foto da galera. 

**Qualquer outra sugestão comentem aqui :grin:**

**Contexto adicional:**

![WhatsApp Image 2021-02-20 at 13 43 00](https://user-images.githubusercontent.com/62030386/109067347-29683300-76cd-11eb-985d-e73c7eac77df.jpeg)

**Comment by JoseRenan** [comment]
Ei, pitaco no layout, acho que seria uma boa deixar isso dos membros mais pra baixo, pra deixar mais em cima o foco nos projetos :thinking:

**Comment by FelipeAmorimFerreira** [comment]
Posso ficar com essa issue e desenvolver a página trocando a posição dos membros e dos projetos?

**Comment by huandrey** [comment]
@FelipeAmorimFerreira pode sim! Aproveita e acompanha o canal do portal no servidor do discord da opendev😁
